README.md

```diff
@@ -9,14 +9,20 @@
 
 ## What and why
 
-Application_config is a module for configuring a python application. It uses toml 
-configuration files that are parsed into dataclasses.
+Application_config is a module for configuring a python application. It uses 
+[toml](https://toml.io/en/) configuration files that are parsed into dataclasses.
 This brings some benefits:
 
-- Configuration parameters are typed
-- IDEs will provide helpful hints and completion when using configuration parameters
-- More control over what happens when a config file contains incorrect or malicious text
-- Possibility to specify defaults when no config file is found
+- Configuration parameters are typed, which allows for improved static code analyses;
+- IDEs will provide helpful hints and completion when using configuration parameters;
+- More control over what happens when a config file contains mistakes
+  (by leveraging the power of [pydantic](https://docs.pydantic.dev/));
```

Comment:
the first three are ended with a `;` not with a `.` is that on purpose?

Reply:
It was on purpose, but I just learned a few things :D
https://taaladvies.net/leestekens-en-hoofdletters-in-een-puntsgewijze-opsomming-algemeen/#:~:text=Na%20de%20leden%20van%20de%20opsomming%20staan%20doorgaans%20puntkomma's%20en,kunnen%20alle%20leestekens%20achterwege%20blijven.